corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -173,7 +173,7 @@ hqDefine('cloudcare/js/backbone/cases.js', function () {
             var self = this,
                 text = self.lookupField(col),
                 td = $("<td/>");
-            if (text) {
+            if (text !== null) {
```

Comment:
can text be undefined, false, or blank string, and if it is, will it be handled correctly?

( <3 javascript )